### ISSUE TITLE
Add face for ace-window

### ISSUE THREAD

**Issue by eflanigan00** [nashamri/spacemacs-theme#97]
What I'm using now:
  ```
(set-face-attribute 'aw-leading-char-face nil
                      :weight 'bold
                      :slant 'italic
                      :foreground "#a31db1"   ;; "#4f97d7"
                      :background "#293239")

```

**Comment by nashamri** [comment]
Thanks @eflanigan00 just pushed an update to support ace-window. You can close this issue.

**Comment by nashamri** [comment]
I'm closing this now, thanks @eflanigan00, feel free to comment back if you think there's something wrong with the update.

**Comment by eflanigan00** [comment]
@nashamri awesome box on the text, thanks for adding it!

6975915e3bc23dd961c251c7c7f4f9cf3b2fe8dc

**Comment by eflanigan00** [comment]
Closes 6975915e3bc23dd961c251c7c7f4f9cf3b2fe8dc
